### ISSUE TITLE
problem? loading ICVL test_seq dataset 

### ISSUE THREAD

**Issue by WeihongM** [moberweger/deep-prior-pp#10]
Hi,   @moberweger 
when I want to rerun the main_icvl_com_refine.py, I find the code confusing here.
`
    Seq2 = di.loadSequence('test_seq_1', docom=True) 
    testSeqs = [Seq2]
` L55
why only load the test_seq_1.txt, I do not know why not load the file seq_2.txt , what is the difference between this two files?

**Comment by moberweger** [comment]
Of course you can run the second file as well, there is no difference, it is another test sequence, however very similar.

**Comment by WeihongM** [comment]
hello, @ @moberweger 
now I met something confusing after reading the code,  
in the file main_icvl_com_refine.py row147
`    poseNetParams = ScaleNetParams(type=1, nChan=nChannels, wIn=imgSizeW, hIn=imgSizeH, batchSize=batchSize,
                                   resizeFactor=2, numJoints=1, nDims=3)`
why you set numJoints = 1 rather than 16?

**Comment by WeihongM** [comment]
hello, @ @moberweger 
now I met something confusing after reading the code,  
in the file main_icvl_com_refine.py row147
`    poseNetParams = ScaleNetParams(type=1, nChan=nChannels, wIn=imgSizeW, hIn=imgSizeH, batchSize=batchSize,
                                   resizeFactor=2, numJoints=1, nDims=3)`
why you set numJoints = 1 rather than 16?

**Comment by WeihongM** [comment]
whether it means your pretrained model is trained on the only first joint coordinate? 
In my opinion I think this is not quite proper. @moberweger

**Comment by moberweger** [comment]
@WeihongM L147 that you mentioned is only for the refinement of the hand localization (com). Therefore, we only predict a single joint, ie the hand center. Thus numJoints=1 and nDims=3 for the 3D offset.

**Comment by WeihongM** [comment]
@moberweger        sorry, so which file is the code to predict the all numjoints? or is it right to just change the numjoints = 16 to train the network, if not,  can you give me some advice how to write?
and Now I am more confused, in this repo, you use the multi-scale input to train the offset of a single joint, but in the paper deep prior++, the offset is not trained in this way.
what is more, I find this code(L147)  is also in the deep prior repo code, you also use numjoint=1 and multi-scale input to train. 
I guess you just want to give the first joint to check the loss and error in this code?
Can you understand what I mean, hopefully look for solutions, thx

**Comment by moberweger** [comment]
You should take a look at `main_icvl_posereg_embedding.py` if you want to predict all joints.
Regarding the multi-scale input, it actually does not matter in terms of accuracy. It is the same code from DeepPrior. If you read the paper, you can see the usage of the different networks.

**Comment by WeihongM** [comment]
ok, @moberweger 
Now I try changing the code (change the numJoints from 1 to 16) to check the performance, because in the paper deep prior, you have mentioned this network to predict all the joint coordinate.
 Do you mean the multi-scale input does not have much improvement on the joint accuracy?

**Comment by moberweger** [comment]
The script `main_icvl_posereg_embedding.py` does already what you want, I guess. Also, multi-scale does not help much, yes.

**Comment by WeihongM** [comment]
@moberweger  hello, 
when I run the main_icvl_posereg_embedding.py on the NYU dataset, I got the mean error 13.8069019318mm, but in the paper deep prior ++ which is 12.3 mm on the NYU dataset.  Can you give me some advice on improving?

**Comment by moberweger** [comment]
main_icvl_posereg_embedding.py is intended for ICVL dataset. So please check that. Did you use the refinement network for the hand localization?

**Comment by WeihongM** [comment]
thanks for your reply very, i know what i missed
